app/controllers/idv/link_sent_controller.rb

```diff
@@ -7,7 +7,7 @@ class LinkSentController < ApplicationController
     include IdvStepConcern
     include StepIndicatorConcern
 
-    before_action :confirm_not_rate_limited
+    before_action :confirm_not_rate_limited, except: [:update]
```

Comment:
i think this is fine for reasons similar to https://github.com/18F/identity-idp/pull/9370/files#r1357329348